packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
-import React from "react";
+import React, { useRef } from "react";
```

Comment:
Nit: I would recommend removing the `useRef` import, and just use `React.useRef` in the code below, since this is what we do for `useEffect`, etc.